src/ape/utils.py

```diff
@@ -45,10 +53,24 @@
 
 @lru_cache(maxsize=None)
 def get_distributions():
+    """
+    Gives archived packages and modules based on release.
+    """
     return packages_distributions()
 
 
 def is_relative_to(path: Path, target: Path) -> bool:
+    """
+    Searches a path and determines its relevancy.
+    Must overrride path or else will trigger a ``ValueError``.
```

Comment:
Don't include this part --- this is not true.

We don't raise any errors in this method.

We except a `ValueError` to return `False`